content/psy_recommend/psy_recommend_1.md

```diff
@@ -0,0 +1,15 @@
+title: Находите время для
+slug: psy1
```

Comment:
`slug` name is a part of page's url. It's rather unacceptable to have this kind of dumb `slug` name!